### ISSUE TITLE
sudo: semaphore : command not found

### ISSUE THREAD

**Issue by ItN0wPompelup** [ansible-semaphore/semaphore#1909]
Hello how are you,
Today I try to install semaphore but I have a problem at the starting of installation.

Once I have installed semaphore by snap on my ubuntu, I must create a user, so I use the command:
**sudo semaphore user add --admin \
--login john\
--name=John\
--email=john1996@gmail.com \
--password=12345**

return error: **sudo: semaphore: command not found**

if I use
**semaphore user add --admin \
--login john\
--name=John\
--email=john1996@gmail.com \
--password=12345**

return error: **Use sudo to execute semaphore commands.**

I tried with multiple accounts, user, root it doesn't work. After doing multiple searches on the internet without success, I just asked the source directly. Has anyone encountered this problem before? Do you have a solution?

Thank you for reading and your future responses.

Pompelup

![image](https://github.com/semaphoreui/semaphore/assets/162558629/858556d7-4885-453b-964a-00e8a4cce10e)

**Comment by fiftin** [comment]
@ItN0wPompelup what Linux distrib do you use?

**Comment by ItN0wPompelup** [comment]
Ubuntu 22.04.4
Thank you for your answer

**Comment by fiftin** [comment]
@ItN0wPompelup

It is on my Ubuntu server: https://asciinema.org/a/s6etRkZCN3MwiHmyTZnVc1ywD

Try to reinstall Semaphore.

**Comment by ItN0wPompelup** [comment]
OK thanks for the info @fiftin but I already tried to reinstall via snap. I will try to install it via a wget/apt otherwise I will try on another OS version.

**Comment by bodzio4749** [comment]
> OK thanks for the info @fiftin but I already tried to reinstall via snap. I will try to install it via a wget/apt otherwise I will try on another OS version.

You do not need ```sudo``` if you are running as root.  Having said that,  I tried the ```snap``` installation and was confused with Ansible's created environment. I decided to go back to my docker installation, which works fine now after creating my own version of dockerfile.

**Comment by ItN0wPompelup** [comment]
This does not help resolve the problem but by going through a manual installation I successfully installed it without any problem.

**Comment by lCedl** [comment]
Hi all

I've tried today the installation of Semaphore on my RedHat Enterprise 9.3
I installed snap as in the officall instructions and then sudo snap install semaphore works like butter. 
 
 Semaphore is reachable on the machine with localhost:3000 but as soon as I like to create a new user I get the error: 
 `sudo: semaphore: command not found` 
 and without sudo:
 `Use sudo to execute semaphore commands.`
 
 Do someone have a clue what could be the problem?
 I already tired to reinstall the whole thing with snap. Maybe someone have a solution otherwise I install semaphore manual...
 
 Thanks
